src/lib/plot/contour.typ

```diff
@@ -5,11 +5,17 @@
 // Find contours of a 2D array by using marching squares algorithm
 //
 // - data (array): 2D float array (rows => columns)
-// - offset (float): Value (offset) a cell must be greater or equal to to count as true
+// - offset (float): Z value (offset) of a cell compare with `op` to, to count as true
```

Comment:
This isn't clear what offset does. Does it get added to all Z values? Is it a threshold?

Reply:
It is a threshold, I'll add that.